.changeset/thirty-knives-learn.md

```diff
@@ -2,4 +2,4 @@
 "@salt-ds/countries": patch
 ---
 
-Fixed LazyCountrySymbol crash with invalid code
+Fixed LazyCountrySymbol crash with invalid code, and `sharp` was not working correctly.
```

Comment:
What sharp ever supported?

Reply:
It's in the prop, and in `lazyMap`

Reply:
`lazyMap` is auto generated and the props extend CountrySymbolProps where it was intentionally added.

Reply:
Happy to keep it as a fix though